<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>File_Widget</class>
 <widget class="QWidget" name="File_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-190</x>
     <y>560</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="back_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>61</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/backcaption.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/start_button.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>301</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;
border-image: url(:/backg.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>54</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>题目</string>
   </property>
  </widget>
  <widget class="QComboBox" name="choose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="choose_label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QToolButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;黑体&quot;;
border-image: url(:/add_background.png);</string>
   </property>
   <property name="text">
    <string>        修改文件</string>
   </property>
  </widget>
  <widget class="QToolButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;黑体&quot;;
border-image: url(:/delete_background.png);</string>
   </property>
   <property name="text">
    <string>        修改密码</string>
   </property>
  </widget>
  <widget class="QToolButton" name="file_choose">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>39</y>
     <width>25</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="backgroun_picture">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>351</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/file.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="file_skim">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;黑体&quot;;
border-image: url(:/add_background.png);</string>
   </property>
   <property name="text">
    <string>        查看文件</string>
   </property>
  </widget>
  <widget class="QToolButton" name="file_output">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;黑体&quot;;
border-image: url(:/delete_background.png);</string>
   </property>
   <property name="text">
    <string>        全体导出</string>
   </property>
  </widget>
  <zorder>choose_label</zorder>
  <zorder>backgroun_picture</zorder>
  <zorder>label</zorder>
  <zorder>back_label</zorder>
  <zorder>back</zorder>
  <zorder>text</zorder>
  <zorder>title</zorder>
  <zorder>title_label</zorder>
  <zorder>choose</zorder>
  <zorder>add_button</zorder>
  <zorder>delete_button</zorder>
  <zorder>file_choose</zorder>
  <zorder>file_skim</zorder>
  <zorder>file_output</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
